pyccl/bcm.py

```diff
@@ -1,6 +1,7 @@
 from .base import unlock_instance
 from .baryons import BaryonsSchneider15
-from .pyutils import deprecated, check
+from .base import deprecated
+from .pyutils import check
```

Comment:
You are using `check(status, cosmo)` but `cosmo` isn't used at the C-level, so it won't carry any error messages even if there are. Remove `cosmo` (although this is now deprecated so we shouldn't bother).